Tasks/VsTest/vstest.ts

```diff
@@ -49,6 +49,7 @@ try {
     var useNewCollectorFlag = tl.getVariable('tia.useNewCollector');
     var isPrFlow = tl.getVariable('tia.isPrFlow');
     var vsTestVersionForTIA: number[] = null;
+	var ignoreVstestFailure: string = tl.getVariable("vstest.ignoretestfailures");
```

Comment:
Why is a task directly using a variable?  Tasks operate on inputs and users can map variables into inputs.  If you want this an an option it should be a bool input and ideally in the advanced group.